### PR TITLE
Add XFI space

### PR DESCRIPTION
**Pull request by xfinance**
**Add space for XFI**  
  
Etherscan: https://etherscan.io/token/0x5BEfBB272290dD5b8521D4a938f6c4757742c430
CoinGecko: https://www.coingecko.com/en/coins/xfinance
Website: https://xfinance.io
Twitter: https://twitter.com/xfinance_io

### CODE REVIEW

File: spaces/xfinance/index.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "decimals": 18,
+  "symbol": "XFI",
+  "defaultView": "all",
+  "token": "0x5BEfBB272290dD5b8521D4a938f6c4757742c430",
+  "core": [
+    "0xF104f1d4F71920334DC765aD2194B8F450567DF2"
+  ],
+  "min": 20,
+  "invalid": [],
+  "skin": "xfi",
```
Comment (e-asia) [inline spaces/xfinance/index.json:13]: I you want to use the `aave` skin, add it on your skin field.
```suggestion
  "skin": "aave",
```

File: skins/xfi.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  --border-color: #d1d5da;
+  --header-bg: #f7f7f7;
+  --block-bg: white;
+
+  #topnav {
+    border-bottom: 4px solid black !important;
+    border-image-slice: 1;
+    border-image-source: linear-gradient(to left, #b6509e, #2ebac6);
+  }
+}
```
Comment (e-asia) [inline skins/xfi.scss:16]: If you want to use the aave skin please delete the scss file.

**Review by e-asia — CHANGES_REQUESTED** [review]

File: skins/xfi.scss (hunk 2 above)
Comment (xfinance) [reply skins/xfi.scss:16]: sorry, thanks

**Comment by xfinance** [comment]
Done

**Review by e-asia — APPROVED** [review]

**Pull request merged**